src/app/collection-page/collection-page.component.ts

```diff
@@ -121,4 +122,18 @@ export class CollectionPage implements OnInit, OnDestroy {
   // private updateSearchInRes(value: boolean): void{
   //  this.searchInResults = value;
   // }
+
+  public skipToFilterSec(): void{
+    window.setTimeout(() => {
+      let htmlelement = this._dom.byId('skip-to-search-link');
+      htmlelement.focus();
+    }, 100);
+  }
+
+  public skipToSearchSec(): void{
```

Comment:
also also here

Reply:
@willschoener  updated in the next commit `dd77c08`.